pkg/k8sutils/job.go

```diff
@@ -127,12 +126,26 @@ func (k8s *k8sImpl) CreateK8sJob(
 							},
 							Env: []v1.EnvVar{
 								{
-									Name:  "CONFIGURATION_SERVICE",
-									Value: jobSettings.InitContainerConfigurationServiceAPIEndpoint,
+									Name: "CONFIGURATION_SERVICE",
+									ValueFrom: &v1.EnvVarSource{
+										ConfigMapKeyRef: &v1.ConfigMapKeySelector{
+											LocalObjectReference: v1.LocalObjectReference{
+												Name: "job-service-config",
+											},
+											Key: "init_container_configuration_endpoint",
+										},
+									},
```

Comment:
This means that the underlying job needs to be able to access the configmap - I'm not sure if we want that

Reply:
Confirmed, this works but doesn't provide the underlying job access to the kubernetes api, so everything works nicely